### DISCUSSION TITLE
"Suggested plugins" vs "Others"

### DISCUSSION THREAD

**Discussion by Pierre-de-la-Verre** [WinMerge/winmerge#1139] [Q&A]
Is there a relevant difference between sub- and sub-sub-menu?
![grafik](https://user-images.githubusercontent.com/13277944/149815488-5689b6fa-fd36-4c96-848b-2976c7962c20.png)

**Comment by sdottaka** [comment]
"Suggested Plugins" shows plugins that are related to the extension of the currently opened files.
"Others" shows plugins that are not in the category such as "Content Extraction" and "Prettification". 

The Unpacker menu will help you understand that.

**Comment by Pierre-de-la-Verre** [comment]
Thanks
but in this case it is confusing. "Other plugins - Others" contains also the plugins which are "suggested". So "Others" contains in reality "All" plugins. Do I see it correct?

**Reply by sdottaka** [reply]
It might be better to change "Other plugins" to "All plugins"

**Comment by Pierre-de-la-Verre** [comment]
Fine. OK.

**Comment by Gitoffthelawn** [comment]
Simple enough (hopefully).  PR: https://github.com/WinMerge/winmerge/pull/1140
